distributed.go

```diff
@@ -153,6 +157,16 @@ func (h Handler) syncDistributedRead(ctx context.Context) error {
 			continue
 		}
 
+		if h.Distributed.PurgeAge != 0 && state.Timestamp.Before(now().Add(-time.Duration(h.Distributed.PurgeAge))) {
```

Comment:
Stupid question, but do you think this would ever delete its own state file? :smile:

I'm not really worried about it at this point either way, but just a thought I had.

Reply:
Fortunately it won't, because there's a check at the top of the loop for the local instance's filename. Thus, this wouldn't be reached.

Reply:
Oh yeah, duh :sweat_smile: